packages/create-bison-app/template/package.json.ejs

```diff
@@ -110,13 +96,11 @@
     "jest": "^27.2.1",
     "jest-environment-node": "^27.2.0",
     "lint-staged": "^10.2.11",
-    "msw": "^0.20.1",
```

Comment:
Not used anywhere in the codebase

Reply:
We may want to use something like this to mock NextAuth later. 
But agree -- remove for now and we can determine what's needed.